perses/app/relative_setup.py

```diff
@@ -934,89 +635,119 @@ def __init__(self, topology_proposal, pos_old, new_positions, use_dispersion_cor
             all water.
-        scheduler_address : str, default None
-            The address of the dask scheduler. If None, local will be used.
-        eq_splitting_string : str, default V R O R V
+        eq_splitting_string : str, default 'V R O R V'
             The integrator splitting to use for equilibrium simulation
-        neq_splitting_string : str, default V R O H R V
-            The integrator splitting to use for the nonequilibrium simulation
+        neq_splitting_string : str, default 'V R O R V'
+            The integrator splitting to use for nonequilibrium switching simulation
+        neglected_new_angle_terms : list, default []
+            list of indices from the HarmonicAngleForce of the new_system for which the geometry engine neglected.
+            Hence, these angles must be alchemically grown in for the unique new atoms (forward lambda protocol)
+        neglected_old_angle_terms : list, default []
+            list of indices from the HarmonicAngleForce of the old_system for which the geometry engine neglected.
+            Hence, these angles must be alchemically deleted for the unique old atoms (reverse lambda protocol)
+        ncmc_save_interval : int, default None
+            interval with which to write ncmc trajectory.  If None, trajectory will not be saved.
+            We will assert that the ncmc_nsteps % ncmc_save_interval = 0; otherwise, the protocol will not be complete
+        write_ncmc_configuration : bool, default False
+            whether to write ncmc annealing perturbations; if True, will write every ncmc_save_interval iterations
         """
-        if scheduler_address is None:
-            self._map = map
-            self._gather = lambda mapped_list: list(mapped_list)
-        else:
-            if scheduler_address == 'localhost':
-                self._client = distributed.Client()
-            else:
-                self._client = distributed.Client(scheduler_address)
-            self._map = self._client.map
-            self._gather = self._client.gather
+        #Specific to LSF clusters
+        # NOTE: assume that the
+        _logger.debug(f"instantiating NonequilibriumSwitchingFEP...")
 
         # construct the hybrid topology factory object
-        self._factory = HybridTopologyFactory(topology_proposal, pos_old, new_positions,
-                                              use_dispersion_correction=use_dispersion_correction)
+        _logger.info(f"writing HybridTopologyFactory")
+        self._factory = HybridTopologyFactory(topology_proposal, pos_old, new_positions, neglected_new_angle_terms, neglected_old_angle_terms)
+        self.geometry_engine = geometry_engine
+        self._ncmc_save_interval = ncmc_nsteps if not ncmc_save_interval else ncmc_save_interval
+        _logger.debug(f"ncmc save interval set as {self._ncmc_save_interval}")
+
+        #we have to make sure that there is no remainder from ncmc_nsteps % ncmc_save_interval
+        try:
```

Comment:
Could this be a warning instead? 

>>> if ncmc_nsteps % self._ncmc_save_interval !=0:
    #set ncmc_save_interval to the nearest number that does fit this requirement, and throw a warning?